### PR TITLE
Release 11.3.18

### PR DESCRIPTION
**Pull request by camscale**

### CODE REVIEW

**Comment by camscale** [comment]
I've updated this to include https://github.com/gravitational/teleport/pull/29582 after merging it.

File: CHANGELOG.md
Comment (tcsc) [inline CHANGELOG.md:5]: Optional: I'd consider trimming the changelog down a bit to user-focussed changes. Things like changing the Go version are possibly a bit developer centric, but it's not enough for me to block the merge.

**Review by tcsc — APPROVED** [review]

File: CHANGELOG.md
Comment (camscale) [reply CHANGELOG.md:5]: I included that because it was included in previous releases. I suspect so that people who track vulnerabilities can know we're building with a version without a vulnerability?

**Review by r0mant — APPROVED** [review]

**Comment by zmb3** [comment]
Might want to ensure `e` ref is up to date before cutting the release, too.

**Comment by camscale** [comment]
I've rebased the release/11.3.18 branch to include https://github.com/gravitational/teleport/pull/29575 which was merged after I branched. It is a performance improvement which I don't think warrants an entry in the changelog, so I'll just silently bring in the change.

I've also updated the `e` ref in this PR as requested by @zmb3.

**Pull request merged**
